packages/driver/test/cypress/integration/commands/actions/type_spec.coffee

```diff
@@ -3037,9 +3037,21 @@ describe "src/cy/commands/actions/type", ->
 
       cy.get("#input-covered-in-span").clear({timeout: 1000, interval: 60})
 
-    it "works on input[type=number]", ->
-      cy.get("#number-with-value").clear().then ($input) ->
-        expect($input.val()).to.equal("")
+    context "works on input type", ->
+      inputTypes = [
```

Comment:
I'm thinking we should test out [these other input types we support](https://docs.cypress.io/api/commands/type.html#Supported-Elements) for `cy.type()`.